Makefile

```diff
@@ -48,11 +48,7 @@ asset-page-flag: ## insert a waffle flag into local docker devstack
 publish: publish-patch ## default is a path release
 
 publish-%: ## publish a new version from master. Argument will be fed into `npm version`, see https://docs.npmjs.com/cli/version for valid values.
-	CURRENT_BRANCH=$$(git rev-parse --abbrev-ref HEAD)
-	if [ "$$CURRENT_BRANCH" != "master" ]; then echo "you may only publish from master" && exit 1; fi
+	if [ $$(git rev-parse --abbrev-ref HEAD) != "master" ]; then echo "you may only publish from master" && exit 1; fi
 	@git diff --quiet || (echo 'unclean git repo, please commit all changes before publish'; exit 1)
-	VERSION=$$(npm version patch)
-	git checkout -b dahlia/$VERSION
-	npm publish --access public
-	git push --set-upstream origin dahlia/$$VERSION && git push --tags
+	export VERSION=$$(npm version patch) && git checkout -b dahlia/$$VERSION && npm publish --access public && git push --set-upstream origin dahlia/$$VERSION && git push --tags
```

Comment:
TIL that it's hard to define local variables in Makefiles, because each line in a recipe gets a new shell. Thus, I smashed multiple things down into a single line